annotation-processor/src/main/java/org/fulib/fx/FxClassGenerator.java

```diff
@@ -51,7 +45,8 @@ public class FxClassGenerator {
      */
     private final TypeMirror parent;
 
-    public FxClassGenerator(ProcessingEnvironment processingEnv) {
+    public FxClassGenerator(ControllerAnnotationProcessor processor, ProcessingEnvironment processingEnv) {
```

Comment:
I generally like this refactoring, but I believe we should have a third class to not overgrow the ControllerAnnotationProcessor class (which, going by the name, already does way more than it should).